R/tests.R

```diff
@@ -136,7 +136,8 @@ renv_tests_diagnostics <- function() {
 
 # Cache absolute path to tests/testthat
 renv_tests_root <- function() {
-  global("tests.root", normalizePath(testthat::test_path(".")))
+  the$tests.root <- the$tests.root %??% normalizePath(testthat::test_path("."))
+  the$tests.root
```

Comment:
This line might not be necessary? (The above should invisibly return the assigned value IIUC)

Reply:
It's not strictly necessary, but I think it's weird for `renv_test_root()` to return invisibly since you're not calling it for its side effect.

Maybe a clearer/simple change would be to wrap the assignment in `()`?

Reply:
Ah, I hadn't thought about visibility of a return value as a signal that the function is used for its side effect.

Wrapping with `()` seems okay.